helix-term/src/application.rs

```diff
@@ -184,6 +186,67 @@ impl Application {
+                                    let main = debugger
+                                        .threads()
+                                        .await
+                                        .ok()
+                                        .and_then(|threads| threads.get(0).cloned());
+                                    if let Some(main) = main {
+                                        let (bt, _) = debugger.stack_trace(main.id).await.unwrap();
+                                        debugger.stack_pointer = bt.get(0).cloned();
+                                    }
+
+                                    let body: helix_dap::events::Stopped = from_value(ev.body.expect("`stopped` event must have a body")).unwrap();
+                                    let scope = match body.thread_id {
+                                        Some(id) => format!("Thread {}", id),
+                                        None => "Target".to_owned(),
+                                    };
+
+                                    let mut status = format!("{} stopped because of {}", scope, body.reason);
+                                    if let Some(desc) = body.description {
+                                        status.push_str(&format!(" {}", desc));
+                                    }
+                                    if let Some(text) = body.text {
+                                        status.push_str(&format!(" {}", text));
+                                    }
+                                    if body.all_threads_stopped == Some(true) {
+                                        status.push_str(" (all threads stopped)");
+                                    }
+
+                                    self.editor.set_status(status);
+                                    self.render();
+                                }
+                                "output" => {
+                                    let body: helix_dap::events::Output = from_value(ev.body.expect("`output` event must have a body")).unwrap();
+
+                                    let prefix = match body.category {
+                                        Some(category) => format!("Debug ({}):", category),
+                                        None => "Debug:".to_owned(),
+                                    };
+
+                                    self.editor.set_status(format!("{} {}", prefix, body.output));
+                                    self.render();
+                                }
+                                "initialized" => {
+                                    self.editor.set_status("Debugged application started".to_owned());
+                                    self.render();
+                                }
+                                _ => {}
+                            }
+                        },
+                        Payload::Response(_) => unreachable!(),
+                        Payload::Request(_) => todo!(),
```

Comment:
The only reverse request debug adapter can send to editor is `runInTerminal`

We can handle this by creating a dialog window asking user to run a command in a terminal and then press enter. Also we can probably make some platform-specific `runInTerminal` implementation.